packages/constants/src/constants.ts

```diff
@@ -303,14 +303,25 @@ export const STORAGE_KEYS = {
   LOGIN_STRATEGY: 'loginStrategy',
 } as const;
 
+export const HTTP_SECURITY_HEADER_KEYS = {
+  'Content-Security-Policy': 'Content-Security-Policy',
+  'Referrer-Policy': 'Referrer-Policy',
+  'Permissions-Policy': 'Permissions-Policy',
+  'Strict-Transport-Security': 'Strict-Transport-Security',
+  'X-XSS-Protection': 'X-XSS-Protection',
+  'X-Content-Type-Options': 'X-Content-Type-Options',
+  'X-Frame-Options': 'X-Frame-Options',
+} as const;
+export type THttpSecurityHeaders = keyof typeof HTTP_SECURITY_HEADER_KEYS;
 export const HTTP_SECURITY_HEADERS = {
-  'Referrer-Policy': 'same-origin',
-  'Permissions-Policy':
-    'microphone=(), camera=(), payment=(), usb=(), geolocation=()',
-  'Strict-Transport-Security': 'max-age=31536000; includeSubDomains; preload',
-  'X-XSS-Protection': '1; mode=block',
-  'X-Content-Type-Options': 'nosniff',
-  'X-Frame-Options': 'SAMEORIGIN',
+  [HTTP_SECURITY_HEADER_KEYS['Referrer-Policy']]: 'same-origin',
+  [HTTP_SECURITY_HEADER_KEYS['Permissions-Policy']]:
+    'microphone=(self), camera=(self), payment=(self), usb=(self), geolocation=(self)',
```

Comment:
Changing the defaults to use `(self)`.

For reference: this is important to ensure that Custom Views (rendered within an `iframe`) can correctly use and override the `Permissions-Policy` header.

Reply:
Perhaps we can add this as a code comment for future reference

Reply:
Good idea, thanks! https://github.com/commercetools/merchant-center-application-kit/pull/3696/commits/d062758b7436ada8d2a2141be501712041118f8a